src/opa.js

```diff
@@ -22,23 +22,28 @@ function stringDecoder(mem) {
  * Stringifies and loads an object into OPA's Memory
  * @param {WebAssembly.Instance} wasmInstance
  * @param {WebAssembly.Memory} memory
- * @param {any} value
+ * @param {any | ArrayBuffer} value
```

Comment:
[nit] `any | X` always is equivalent to `any`, isn't it? But it makes sense to keep this for documenting the feature -- would it make sense to add a brief note after `value`, though, to explain what'll happen when it's an ArrayBuffer?